test/WebSites/ModelBindingWebSite/Models/BindingBehaviorModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.ModelBinding;
+
+namespace ModelBindingWebSite
+{
+    public class BindingBehaviorModel
+    {
+        [BindingBehavior(BindingBehavior.Never)]
+        public string Property1 { get; set; }
+
+        [BindingBehavior(BindingBehavior.Optional)]
+        public string Property2 { get; set; }
+
+        [BindingBehavior(BindingBehavior.Required)]
+        public string Property3 { get; set; }
+
+        [BindRequired]
+        public string Property4 { get; set; }
+
+        [BindNever]
+        public string Property5 { get; set; }
```

Comment:
property names are rather obscure.  suggest `BehaviourNeverProperty`, `BehaviourOptionalProperty`, `BehaviourRequiredProperty`, `BindRequiredProperty`, and `BindNeverProperty`.